base-image/automation/jenkins-build.sh

```diff
@@ -5,21 +5,43 @@ set -o pipefail
 
 date=$(date +'%Y%m%d' -u)
 REPO='resin/amd64-supervisor-base'
+# MACHINE_LIST: qemux86-64 qemux86 generic-armv6 generic-armv7hf generic-armv5
+# MACHINE_LIST should be set in jenkins config
 
 git submodule update --init --recursive
 rm -rf dest
 mkdir dest
 
 docker build -t supervisor-base-builder .
-docker run --rm \
-	-v `pwd`:/source \
-	-v `pwd`/dest:/dest \
-	supervisor-base-builder
-if [ -f dest/rootfs.tar.gz ]; then
-	docker import dest/rootfs.tar.gz $REPO:$date
-	docker tag -f $REPO:$date $REPO:latest
-	docker push $REPO
-else
-	echo "rootfs is missing!"
-	exit 1
-fi
+for machine in $MACHINE_LIST; do
+	case "$device" in
+	'qemux86-64')
+		REPO='resin/amd64-supervisor-base'
+	;;
+	'qemux86')
```

Comment:
The default qemu machines set some properties that are unrelated to containers. Let's create generic machine definitions for the qemux86 and qemux86-64 machines too.

`generic-x86` should only include `conf/machine/include/tune-i586.inc` and `generic-x86-64` should only include `conf/machine/include/tune-core2.inc` and define `DEFAULTTUNE ?= "core2-64".